scipy/fft/tests/test_basic.py

```diff
@@ -5,52 +5,81 @@
 import pytest
 from numpy.random import random
 from numpy.testing import (
-        assert_array_almost_equal, assert_array_equal, assert_allclose
-        )
+    assert_array_almost_equal, assert_array_equal, assert_allclose
+)
 from pytest import raises as assert_raises
 import scipy.fft as fft
+from scipy.conftest import (
+    array_api_compatible,
+    skip_if_array_api,
+    skip_if_array_api_gpu,
+    skip_if_array_api_backend,
+)
+from scipy._lib._array_api import SCIPY_ARRAY_API, as_xparray, array_namespace
 
-def fft1(x):
+
+def fft1(x, xp):
     L = len(x)
-    phase = -2j*np.pi*(np.arange(L)/float(L))
-    phase = np.arange(L).reshape(-1, 1) * phase
-    return np.sum(x*np.exp(phase), axis=1)
+    phase = -2j*xp.pi*(xp.arange(L)/float(L))
+    phase = xp.arange(L).reshape(-1, 1) * phase
+    return xp.sum(x*xp.exp(phase), axis=1)
 
 
 class TestFFTShift:
 
+    @skip_if_array_api
```

Comment:
Shouldn't we include this in the array api tests?

Reply:
Pushed a fix for this.